tutoroars/templates/oars/jobs/init/oars_init_schemas_tables_users.sh

```diff
@@ -31,11 +40,12 @@ PRIMARY KEY (emission_time, event_id);
 CREATE TABLE IF NOT EXISTS {{ OARS_XAPI_DATABASE }}.{{ OARS_XAPI_TABLE }} (
     event_id UUID NOT NULL,
     verb_id String NOT NULL,
-    actor_id UUID NOT NULL,
+    actor_id String NOT NULL,
+    object_id String NOT NULL,
     org String NOT NULL,
     course_id String NOT NULL,
     emission_time DateTime64(6) NOT NULL,
-    event JSON NOT NULL
+    event_str String NOT NULL
```

Comment:
I don't understand how  the JSON data type is poorly supported in Clickhouse? I've been loving how I can dig into it with the SQLExplorer in Superset, and it makes it very easy to dig out event specifics that aren't in their own fields (like whether a problem was answered successfully or not).

Reply:
Ok, sure.. that's still a shame! I had hoped we could avoid many tedious schema changes whenever we want to chart new xAPI event types! But oh well..

I guess anyone who's in a rush can still use the https://clickhouse.com/docs/en/sql-reference/functions/json-functions